org.opent2t.sample.hub.superpopular/com.nest.hub/js/package.json

```diff
@@ -19,10 +19,12 @@
   },
   "homepage": "https://github.com/openT2T/translators#readme",
   "dependencies": {
-    "request": "^2.74.0",
-    "request-promise": "^4.1.1",
+    "firebase": "2.4.2",
     "opent2t": "^1.0.1",
-    "opent2t-onboarding-org-opent2t-onboarding-nest": "^0.0.1"
+    "opent2t-onboarding-org-opent2t-onboarding-nest": "^0.2.0",
```

Comment:
There is a new set of changes for onboarder (like update the ttl) waiting to be reviewed. This translator uses that newer version of translator.

---
In reply to: [98255223](https://github.com/openT2T/translators/pull/146#discussion_r98255223) [](ancestors = 98255223)